### DISCUSSION TITLE
Null reference on Modal.Show

### DISCUSSION THREAD

**Discussion by ghost** [Blazored/Modal#577] [Q&A]
I am new to Blazor  using .Net 8.0 and getting error System.NullReferenceException: 'Object reference not set to an instance of an object.' on Modal.Show.
Instead of using  [CascadingParameter] public IModalService Model { get; set; } = default!;, injecting the service @inject IModalService modal; gets rid of the null reference error but still doesn't show the window. The result from Show method has status "Waiting for Activation".

**Comment by ghost** [comment]
.Net 8 Support solution helped this.

**Reply by wallyhall** [reply]
Thank-you - as far as I can see, none of the docs or examples on this repo mention this.  I had been pulling my hair out.
